core/src/main/java/com/google/cloud/sql/core/SslSocketFactory.java

```diff
@@ -203,6 +211,50 @@ private SSLSocket createAndConfigureSocket(
     return sslSocket;
   }
 
+  /**
+   * Converts the string property of IP types to a list by splitting by commas, and upper-casing.
+   */
+  public static List<String> listIpTypes(String cloudSqlIpTypes) {
```

Comment:
I'm a little uncomfortable with doing validation or making it an enum because it makes it harder to make changes later (particularly for enums, if we introduce a new value or remove one, it would require a new build)